src/main/java/thrift/logic/commands/ListCommand.java

```diff
@@ -14,25 +17,43 @@
 
     public static final String HELP_MESSAGE = COMMAND_WORD
             + ": Lists all transactions or lists transactions filtered by month\n"
-            + "Format: " + COMMAND_WORD + " [" + CliSyntax.PREFIX_MONTH + "MONTH] (must be of format MMM)\n"
+            + "Format: " + COMMAND_WORD + " [" + CliSyntax.PREFIX_MONTH + "MONTH] (must be of format MM/yyyy)\n"
             + "Possible usages of " + COMMAND_WORD + ": \n"
             + "To list all transactions: " + COMMAND_WORD + "\n"
-            + "To list all transactions that were created in January: "
-            + COMMAND_WORD + " " + CliSyntax.PREFIX_MONTH + "jan";
+            + "To list all transactions that were created in January 2019: "
+            + COMMAND_WORD + " " + CliSyntax.PREFIX_MONTH + "01/2019";
 
     public static final String MESSAGE_SUCCESS = "Listed all transactions";
 
+    public static final String MESSAGE_SUCCESS_MONTH_FILTER = "Listed transactions in specified month";
+
     public static final String MESSAGE_USAGE = COMMAND_WORD
             + ": lists all transactions or list transactions filtered by month.\n"
             + "Parameters: "
-            + CliSyntax.PREFIX_MONTH + "MONTH (must be of format MMM)\n"
+            + CliSyntax.PREFIX_MONTH + "MONTH (must be of format MM/yyyy)\n"
             //+ CliSyntax.PREFIX_TAG + "TAG...\n"
-            + "Example: " + COMMAND_WORD + " " + CliSyntax.PREFIX_MONTH + "jan";
+            + "Example: " + COMMAND_WORD + " " + CliSyntax.PREFIX_MONTH + "01/2019";
+
+    private Calendar monthyear;
+
+    public ListCommand(Calendar monthyear) {
+        this.monthyear = monthyear;
+    }
+
+    public ListCommand() {
+        this.monthyear = null;
+    }
 
     @Override
     public CommandResult execute(Model model) {
-        requireNonNull(model);
-        model.updateFilteredTransactionList(Model.PREDICATE_SHOW_ALL_TRANSACTIONS);
-        return new CommandResult(MESSAGE_SUCCESS);
+        if (this.monthyear == null) {
+            requireNonNull(model);
+            model.updateFilteredTransactionList(Model.PREDICATE_SHOW_ALL_TRANSACTIONS);
+            return new CommandResult(MESSAGE_SUCCESS);
+        } else {
+            requireNonNull(model);
+            model.updateFilteredTransactionList(new TransactionIsInMonthYearPredicate(this.monthyear));
```

Comment:
Should create a method `ModelManager#setCurrentMonthYear(Calendar monthYear)` to change the `currentMonthYear` in `ModelManager`. Afterwards simply call `model.updateFilteredTransactionListToCurrentMonth()`

Example:
```
} else {
            model.setCurrentMonthYear(this.monthyear);
            model.updateFilteredTransactionListToCurrentMonth();
            return new CommandResult(MESSAGE_SUCCESS_MONTH_FILTER);
        }
```

NOTE: You will have to add the method signature of `void setCurrentMonthYear(Calendar monthYear)` into `Model`, as well as the ModelStubs in the tests.